src/foam/dao/EasyDAO.js

```diff
@@ -170,12 +172,18 @@ return delegate;
       generateJava: false,
       class: 'Property'
     },
+//    {
+//      /** Enable local in-memory caching of the DAO. */
+//      class: 'Boolean',
+//      name: 'cache',
+//      generateJava: false,
+//      value: false
+//    },
     {
-      /** Enable local in-memory caching of the DAO. */
-      class: 'Boolean',
-      name: 'cache',
-      generateJava: false,
-      value: false
+      class: 'foam.core.Enum',
+      of: 'foam.dao.CacheType',
+      name: 'cacheType',
+      value: 'NONE' /* 'None'*/
```

Comment:
Should it be
```suggestion
      value: foam.dao.CacheType.NONE /* 'None'*/
```

Reply:
Both work because the adapt method for enums will make the conversion.
@mcarcaso Is it better to just do 'NONE' for class loading purposes?

Reply:
Then will leave it as "NONE"